cas-server-support-oauth/src/test/java/org/jasig/cas/support/oauth/web/OAuth20AuthorizeControllerTests.java

```diff
@@ -43,6 +45,8 @@
     private static final String ID = "id";
     private static final String FIRST_NAME_ATTRIBUTE = "firstName";
     private static final String FIRST_NAME = "jerome";
+    private static final String LAST_NAME_ATTRIBUTE = "lastName";
+    private static final String LAST_NAME = "LELEU";
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['VARIABLE_DEF' should be separated from previous statement.](https://www.codacy.com/app/mmoayyed/cas/file/2491108599/issues/source?bid=3256115&fileBranchId=3256131#l49)